lib/node_modules/@stdlib/blas/ext/base/dnanasum/manifest.json

```diff
@@ -35,8 +35,12 @@
 			],
 			"libpath": [],
 			"dependencies": [
+				"@stdlib/napi/export",
+                "@stdlib/napi/argv",
+                "@stdlib/napi/argv-int64",
+                "@stdlib/napi/argv-strided-float64array",
```

Comment:
incorrect indentation

Reply:
I have update the manifest with correct indentation